src/main/java/module-info.java

```diff
@@ -1,7 +1,6 @@
 open module org.jabref {
     // Swing
     requires java.desktop;
-
```

Comment:
Do not remove empty lines if not necessary

Reply:
Done